### PR TITLE
[cuda] Fix find cuda in UNIX

### PR DESCRIPTION
**Pull request by JackBoosY**
Fix find cuda in UNIX.

Related: #8247.

### CODE REVIEW

**Review by NancyLi1013 — APPROVED** [review]

File: ports/cuda/portfile.cmake [hunk 1]
```diff
@@ -2,14 +2,51 @@
...
+        set(platform_base "/Developer/NVIDIA/CUDA-")
+    endif()
+    
+    file(GLOB possible_paths "${platform_base}*")
+    set(FOUND_PATH )
+    foreach (p ${possible_paths})
+        # Extract version number from end of string
+        string(REGEX MATCH "[0-9][0-9]?\\.[0-9]$" p_version ${p})
+        if (IS_DIRECTORY ${p} AND p_version)
+            message("FOUND_PATH : ${p}")
```
Comment (strega-nil) [inline ports/cuda/portfile.cmake:31]:
```suggestion
```

remove the debug message :)

**Comment by strega-nil** [comment]
Alright, awesome! Thanks Jack! 😄

**Pull request merged**